src/app/static/src/app/components/surveyAndProgram/SurveyAndProgram.vue

```diff
@@ -190,6 +191,15 @@
                     options.push({id: "0", label: i18next.t("ANC", lang)});
                 }
                 return options;
+            },
+            defaultDataSource() {
+                const selectDefaultDataset = !this.programme.available && this.anc.available ? "anc" : null
+                const showDataSourcePicker = !!this.anc.available && !!this.programme.available
```

Comment:
```suggestion
                const showDataSourcePicker = this.anc.available && this.programme.available
```
These are sure to be booleans, so removing `!!` is safe and improves readability

Reply:
I know right, sometimes returns undefined if nothing is selected, I just want boolean.